packages/terra-site/src/examples/card/CardPaddingVertical.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Card from 'terra-card';
+
+const CardPaddingVertical = () => (
+  <div>
+    <Card>
+      <Card.Padding hasPaddingVertical>Hello World!!</Card.Padding>
```

Comment:
nit: might be worth one of these examples having multiple children? The use case mike had in mind for the `Card.Padding` component is a card where it has text, then HR, then text and everything except the HR wants padding around the edge (HR's should touch the edges